public/apps/configuration/utils/delete-confirm-modal-utils.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import { EuiOverlayMask, EuiConfirmModal } from '@elastic/eui';
+
+/**
+ *
+ * @param handleDelete: [Type: func] delete function which needs to be execute on click of confirm button.
+ * @param selection: Selected list of items.
+ * @param entity: e.g. roles, tenants, users, mapping etc. This will display in confirmation text before deletion.
+ * @param customText: If you want other than default confirm message, pass it as customText.
+ */
+export function useDeleteConfirmState(
+  handleDelete: () => Promise<void>,
+  selection: any[],
```

Comment:
If we just need the count, I would suggest just pass the count as parameter.

Reply:
Fixed. Passing count as parameter.